src/drivers/distance_sensor/lightware_sf45_serial/lightware_sf45_serial.cpp

```diff
@@ -762,10 +674,63 @@ void SF45LaserSerial::sf45_process_replies(float *distance_m)
 		break;
 	}
 }
+void SF45LaserSerial::_publish_obstacle_msg(hrt_abstime now)
+{
+	// This whole logic is here because the sensor has a faster loop time than collision prevention, and if we dont work with a timeout on sensor data
```

Comment:
I don't understand the comment, maybe we could quickly chat about it tomorrow @Claudio-Chies